src/System.IO.Pipelines/ThrowHelper.cs

```diff
@@ -52,6 +52,11 @@ public static void ThrowArgumentOutOfRangeException_BufferRequestTooLarge(int ma
             throw GetArgumentOutOfRangeException_BufferRequestTooLarge(maxSize);
         }
 
+        public static void ThrowObjectDisposedException(string objectName)
+        {
+            throw GetObjectDisposedException(objectName);
+        }
+
```

Comment:
Trouble is OwnedBuffer is generic `<T>` which means all T's get there own ThrowHelper as well